### ISSUE TITLE
Make it payable with Libra-cli

### ISSUE THREAD

**Issue by totiz** [kulapio/libra-merchant#12]
Right now we only support [Libra Wallet POC](https://dev.kulap.io/libra), so I think we should allow simple full address and QR of full address that libra-cli users and other wallet can pay with.

**Comment by lucasverra** [comment]
correct me if im wrong but this is not possible today. Your animation of payment receive comes from pinging the internet, not the blockchain.  This way POC wallet informs (via internet) a server, then libramerchant `getBalance` each second 

cf gif here http://recordit.co/nWPylYEHcn 

In order to make it work 

a. getbalance would need to ping API to check on blockchain
b. Blockchain would need to transmit an event ? 

Curious to see next phase of dev ! keep it up :)

**Comment by totiz** [comment]
Correct me if I understand wrong,

You mean we should to reduce interval call to **getBalance** each second, to know that user already paid, am I right?

If so I suggest to add new issue for "Reduce interval checking of paid".

But for this issue, I mean to add new feature for libra-cli, because right now we introduce new QR format that other wallets may don't understand, for example if we show full address and ask user to sending libra coins to address for specific amount, then the rest will work as normally (will successfully show paid).

**Comment by lucasverra** [comment]
> I mean to add new feature for libra-cli, because right now we introduce new QR format that other wallets may don't understand, for example if we show full address and ask user to sending libra coins to address for specific amount, then the rest will work as normally (will successfully show paid).

`{"type":"peer_to_peer_payment","address":"42d0470ba549c63ad90ab3122fca00f1f57d1d21d8ece102548013bb960150cf","amount":"4.5","merchant":"CLASS CAFE"}` indeed you need to get more generic and less custom. Only receiver address should be transmitted. 
If not, you are just closing the ecosystem to your wallet, which is a non-sens to my regard at least. 

So if i send now from a CLI to 42d0470ba549c63ad90ab3122fca00f1f57d1d21d8ece102548013bb960150cf, should i see the same animation ?

**Comment by totiz** [comment]
@lucasverra Yes, you can see the same animation actually you can try by decoding QR and copy address to libra-cli and send with bill amount. QR Format is used to communicate with Wallet to know address and amount need to pay. So if we have one more option on UI side, just show address and amount, that will be more easy for other wallet to pay with it.
